### PR TITLE
Skip stop/shutdown test cases on lso based cluster

### PR DESCRIPTION
**Pull request by Akarsha-rai**
Fixes: #2079

Signed-off-by: Akarsha <akrai@redhat.com>

### CODE REVIEW

**Comment by RazTamir** [comment]
Hi @Akarsha-rai ,
I think we need to isolate this PR just for AWS i3 instances and not all LSO based deployments

**Comment by Akarsha-rai** [comment]
> Hi @Akarsha-rai ,
> I think we need to isolate this PR just for AWS i3 instances and not all LSO based deployments

Sorry, I got confused. Updated such that will skip only on AWS i3 instances
Thank you:)

File: ocs_ci/framework/pytest_customization/marks.py [hunk 1]
```diff
@@ -108,6 +108,12 @@
     reason="Tests are not running on IPI deployed cluster"
 )
 
+awsi3_deployment_required = pytest.mark.skipif(
```
Comment (clacroix12) [inline ocs_ci/framework/pytest_customization/marks.py:111]: If this is skipping tests when it is configured for AWS I3 why is this marker named `awsi3_deployment_required`? Shouldn't it be something like `skipif_aws_i3`?

Comment (Akarsha-rai) [reply ocs_ci/framework/pytest_customization/marks.py]: Agreed, now changed to skipif_aws_i3.

File: ocs_ci/framework/pytest_customization/marks.py [hunk 2]
```diff
@@ -114,6 +114,12 @@
     reason="Test runs ONLY on IPI deployed cluster"
 )
 
+skipif_aws_i3 = pytest.mark.skipif(
+    config.ENV_DATA['platform'].lower() == 'aws'
+    and config.DEPLOYMENT.get('local_storage') is True,
+    reason="Tests are not running on aws i3 instances"
```
Comment (PrasadDesala) [inline ocs_ci/framework/pytest_customization/marks.py:120]: can this skip reason be changed to "Tests will not run on AWS i3"?
WDYT?

Comment (Akarsha-rai) [reply ocs_ci/framework/pytest_customization/marks.py]: changed

**Comment by Akarsha-rai** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1919/testReport/tests.e2e.monitoring.test_monitoring_on_negative_scenarios/TestMonitoringBackedByOCS/

$ run-ci   --cluster-path /home/akarsha/aws-install/may-22/ --cluster-name tdesala-workloads  tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py::TestMonitoringBackedByOCS::test_monitoring_shutdown_and_recovery_prometheus_node --ocsci-conf=conf/deployment/aws/ipi_3az_rhcos_lso_3m_3w.yaml
Collecting Cluster versions
============================================================================ test session starts ============================================================================
platform linux -- Python 3.7.5, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /home/akarsha/automation/ocs-ci, inifile: pytest.ini
plugins: metadata-1.9.0, html-2.1.1, ordering-0.6, logger-0.5.1, reportportal-1.10.0, marker-bugzilla-0.9.4
collected 1 item                                                                                                                                                                                                  

tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py::TestMonitoringBackedByOCS::test_monitoring_shutdown_and_recovery_prometheus_node SKIPPED                                                     [100%]

**Pull request merged**